haskell/cabal
# Warn about missing `default-language`

Closes #9620 

To accomodate GHC language editions (GHC2021, GHC2024, etc.).

---

Include the following checklist in your PR:


---

**QA notes**

- Start with a `.cabal` file with `cabal-version: 3.4` and no `default-language` field.
- Old `cabal check` *will not* complain about missing default language.
- New `cabal check` *will* emit a `no-default-language` warning.
